From d81f051a214735e5e30b7cefeac70f72aa45d18e Mon Sep 17 00:00:00 2001
From: Steev Klimaszewski <steev@kali.org>
Date: Fri, 17 Feb 2023 16:11:51 -0600
Subject: [PATCH 068/173] arm64: dts: lenovo-x13s: Make it hotter

I don't like limiting the cpu so early, so let it get a bit warmer
before doing so.
---
 arch/arm64/boot/dts/qcom/sc8280xp-lenovo-thinkpad-x13s.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sc8280xp-lenovo-thinkpad-x13s.dts b/arch/arm64/boot/dts/qcom/sc8280xp-lenovo-thinkpad-x13s.dts
index 0c845f37dd8d..864c6c8b5c5d 100644
--- a/arch/arm64/boot/dts/qcom/sc8280xp-lenovo-thinkpad-x13s.dts
+++ b/arch/arm64/boot/dts/qcom/sc8280xp-lenovo-thinkpad-x13s.dts
@@ -304,13 +304,13 @@ skin-temp-thermal {
 
 			trips {
 				skin_temp_alert0: trip-point0 {
-					temperature = <55000>;
+					temperature = <60000>;
 					hysteresis = <1000>;
 					type = "passive";
 				};
 
 				skin_temp_alert1: trip-point1 {
-					temperature = <58000>;
+					temperature = <65000>;
 					hysteresis = <1000>;
 					type = "passive";
 				};
-- 
2.43.0

